README.md

```diff
@@ -71,7 +91,7 @@ pip install mlrun
 MLRun requires separate containers for the API and the dashboard (UI).
 You can also select to use the pre-baked JupyterLab image.
 
-To install and run MLRun locally using Docker or Kubernetes, see the instructions in [**hack/local/README.md**](hack/local/README.md).
+To install and run MLRun locally using Docker or Kubernetes, see the instructions in the [**MLRun documentation**]([hack/local/README.md](https://mlrun.readthedocs.io/en/latest/install.html)).
```

Comment:
The new link doesn't work, looks like a typo

Reply:
fixed